rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/aggregate_min_OA_schedule_across_zones.py

```diff
@@ -0,0 +1,31 @@
+from operator import add
+
+
+def aggregate_min_OA_schedule_across_zones(zone_OA_CFM_list_of_schedules):
+    """Each HVAC system can serve multiple zones which can have multiple terminal units serving each zone. In order to obtain an hourly HVAC level OA CFM schedule the OA CFM provided by each zone needs to be aggregated for each hour of the year. This function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system.
+
+    Parameters
+    ----------
+    zone_OA_CFM_list_of_schedules: list of list
+        A list which contains each zone's 8760 OA CFM schedule as a list value.
+
+    Returns
+    -------
+    list
+        This function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system.
+    """
+
```

Comment:
line 25 takes out the first element from `zone_OA_CFM_list_of_schedules`. It is actually a bit dangerous to do so unless you are 100% sure it is not an empty list. To gain the confident on this operation, we shall first, in the argument specify the type:
```python
aggregate_min_OA_schedule_across_zones(zone_OA_CFM_list_of_schedules: list)
```
second, make sure it is not an empty list
```python
assert_(len(zone_OA_CFM_list_of_schedules)>0, 'The provided list of schedules is an empty list.'
```

Reply:
I assumed ```zone_OA_CFM_list_of_schedules``` to be a list of lists. So, I specified ```List[list]``` type hint.